soundfile.py

```diff
@@ -35,7 +57,7 @@
     'genre':       0x10,
 }
 
-_formats = {
+_formats: Final[Dict[Format, int]] = {
```

Comment:
I'm not particularly versed in the whole typing workflow. Why are these defined as `Format` instead of `StrType`? I don't like how the list of all possible strings is now duplicated in two places.

Reply:
Fixed in https://github.com/bastibe/python-soundfile/pull/462/commits/a8a77c1e8155dc029f53aad1d69a7a501a60a671